### ISSUE TITLE
[BITV]: adapt `aria-label`'s text for checkboxes in files list

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#42683]
`aria-label`'s text stays unchanged while checking in- and out checkboxes like "Select the row for Media". 

- description should be more specific like "Select folder "New folder"" or "Select "Mycat.jpeg" file"
- ~description should be changed depending on a chackbox state like "Select folder "New folder"" -> "Deselect folder "New folder""~

![checkbox](https://github.com/nextcloud/server/assets/6078378/63490eb4-24c8-40ab-82f0-8d70c3ac13c7)

**Comment by Pytal** [comment]
As the checked and indeterminate states already provide this information to screen readers, instead of adding this info to the label itself we should switch it to "toggle" and have the screen reader announce the correct states

**Comment by szaimen** [comment]
I think this is missing a dependency update in stable28

**Comment by szaimen** [comment]
For this one
